employee_project/main.py

```diff
@@ -11,7 +11,7 @@ def main():
         if choice == 1:
             os.system("cls")
             print("***************** REGISTER EMPLOYEE DETAILS ******************************\n\n")
-            name = input("Enter Employee Name: ")
+            name = employee_name(input("Enter Employee Name: "))
```

Comment:
Why your adding here also input and employee.py file also adding the same line to get the input from the user.
employee_name(input("Enter Employee Name: "))

Reply:
removed